pipelines/bootstrap.sh

```diff
@@ -111,7 +111,7 @@ function clone_ztp() {
     echo ">>>> Cloning Repository into your local folder"
     echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
     rm -rf ${WORKDIR}
-    git clone https://github.com/rh-ecosystem-edge/ztp-pipeline-relocatable.git -b ${BRANCH} ${WORKDIR}
+    git clone https://github.com/eifrach/ztp-pipeline-relocatable.git -b ${BRANCH} ${WORKDIR}
```

Comment:
please remove your quay account from here...In main we should use the rh-ecosystem-edge account